### ISSUE TITLE
Support the new Microsoft Edge?

### ISSUE THREAD

**Issue by xfq** [w3c/web-roadmaps#447]
https://developer.microsoft.com/en-us/microsoft-edge/status/ now lists the status of web platform features in both Microsoft Edge (Chromium) and Microsoft Edge (EdgeHTML). Should we support both of them?

**Comment by tidoust** [comment]
Both [Can I Use](https://caniuse.com/) and [Browser Compat Data](https://github.com/mdn/browser-compat-data/blob/master/browsers/edge.json) consider that there's just one Edge with a change of engine between version 18 (based on EdgeHTML) and version 79 (based on Chromium). See, e.g. [discussion in Browser Compat Data](https://github.com/mdn/browser-compat-data/issues/5214#issuecomment-561501951). We should probably do the same, and merely update the icon to the new one.

Actual implementation info has not been updated in Browser Compat Data yet but updates are underway, using some automatic transition logic (if it's supported in Chrome, then it's likely supported in Microsoft Edge (Chromium)).

As far as I can tell, the "Microsoft Edge (Chromium)" info on Microsoft Edge Platform Status does not appear in the underlying [status.json file](https://github.com/MicrosoftEdge/Status/blob/production/status.json). Instead, the info seems to be a mere copy of the "status in Chrome". In particular, all features have the same status in Chrome and in Microsoft Edge (Chromium).

All in all, this may mean that we should stop using this `status.json` file (which hasn't been updated in the past 5 months) and either:
1. rely on implementation info from other data sources
2. consider that, for all features, status in Edge (Chromium) is the same as status in Chrome, which may be true today but may become false over time.

I'll give it a try with 1. and see how that affects final implementation info.

**Comment by tidoust** [comment]
Flagging this for September milestone as this seems to trigger some mistakes.

For instance, for "Custom Elements":
- [Chrome Platform Status](https://www.chromestatus.com/feature/4696261944934400) reports "Positive" for Edge, which we translate to "Under consideration"
- [Microsoft Edge Platform Status](https://developer.microsoft.com/en-us/microsoft-edge/status/customelements/?q=custom) says "Supported in Microsoft Edge (Chromium)" but we don't use that info since it's not in the JSON file exported by that status platform
- [Can I use](https://caniuse.com/custom-elementsv1) says "Supported in Edge"
- The [Edge status JSON file](https://raw.githubusercontent.com/MicrosoftEdge/Status/production/status.json) says: "Not Supported".

In the end, we currently "trust" the Edge status file, but that's for the version of Edge not based on Chromium, so that really isn't the right info to look at. Chrome Platform Status is not up-to-date on that one either. Can I use is correct.

That probably means we should prefer Can I use over other sources for Microsoft Edge status, at least for the time being.

**Comment by xfq** [comment]
Another issue is that only the data for the desktop edge are displayed, but not the mobile versions of Edge.

See the icons in https://w3c.github.io/web-roadmaps/mobile/graphics.html for examples.

**Comment by tidoust** [comment]
True. I had not really considered it because the previous version no longer really existed on mobile devices, but users can indeed now install the new Edge on Android.

There may be more differences between Edge and Chrome on mobile devices though, and platform status web sites do not report anything on Edge for mobile (no dedicated property in Can I use, Chrome platform status,  MDN browser compat data, etc.). I think that we should just stick to desktop as a result.

**Comment by xfq** [comment]
Yeah, and the iOS version uses WebKit instead of Blink, as with all other iOS web browsers.
